lib/commands.js

```diff
@@ -191,9 +191,11 @@ module.exports = async (message, command, args) => {
         }
       }).then(res => res.json()).catch(log.debug)
 
-      let response = `${value} is not a registered GitHub user.`
+      let response = `${user} is not a registered GitHub user.`
```

Comment:
This default is functionally useless now.

Reply:
Good point. The code assumes this user will always exist (since their repo had to exist in order to be merged to phub) so this can be axed.